packages/frontend/src/pages/CreateEventPage/CreateEventPage.tsx

```diff
@@ -1,17 +1,170 @@
+  const [gender, setGender] = useState('');
+  const [eventLanguage, setEventLanguage] = useState('');
+  const [volunteersQuantity, setVolunteersQuantity] = useState('');
+  const [telegramChannelLink, setTelegramChannelLink] = useState('');
+  const [eventStatus, setEventStatus] = useState(EventStatus.Draft);
+
+  const handleCreateEvent = () => {
+    navigate(EVENTS_ROUTE);
+  };
+
+  const handleSubmit = (event: { preventDefault: () => void }) => {
+    event.preventDefault();
+
+    setIsCreating(() => true);
+
+    handleCreateEvent();
+  };
 
   useEffect(() => {
     editor.role ? setIsEditorHasPermissions(() => true) : navigate(ROOT_ROUTE);
   }, [editor, navigate]);
 
-  return isEditorHasPermissions && <div>Create Event Page</div>;
+  return (
+    isEditorHasPermissions && (
+      <div className={css.createEventWrapper}>
+        <div className={css.createEventTitle}>{translation.createEvent}</div>
+        <form className={css.createEventForm} onSubmit={handleSubmit}>
+          <Input
+            value={eventName}
+            setChange={setEventName}
+            labelText={translation.title}
+            required
+          />
+          <Input
+            value={eventDescription}
+            setChange={setEventDescription}
+            labelText={translation.description}
+            required
+          />
+          <Input
+            value={eventPlace}
+            setChange={setEventPlace}
+            labelText={translation.place}
+            required
+          />
+          <Input
+            value={eventStartDate}
+            setChange={setEventStartDate}
+            type='date'
+            labelText={translation.eventStartDate}
+            required
+          />
+          <EventTimeDuration
+            eventStartTime={eventStartTime}
+            setEventStartTime={setEventStartTime}
+            eventEndTime={eventEndTime}
+            setEventEndTime={setEventEndTime}
+          />
+          <Input
+            value={eventEndDate}
+            setChange={setEventEndDate}
+            type='date'
+            labelText={translation.eventEndDate}
+            required
+          />
+          <Input
+            value={eventDuration}
+            setChange={setEventDuration}
+            type='number'
+            labelText={translation.timeDuration}
+            required
+          />
+          <Input
+            value={eventRegistrationDate}
+            setChange={setEventRegistrationDate}
+            type='date'
+            labelText={translation.registrationPeriod}
+            required
+          />
+          <Select
+            value={gender}
+            setChange={setGender}
+            options={VOLUNTEER_GENDER}
+            labelText={translation.gender}
+            required
+            placeholder={translation.choice}
+            selectClassName={!gender ? css.selectPlaceholder : undefined}
+          />
+          <Select
```

Comment:
This should be multiselect, will it work?

Reply:
I will fix it